public/assets/scss/globals/_boilerplate.scss

```diff
@@ -1,5 +1,5 @@
 @import "fonts";
-
+@import "../../../../node_modules/bootstrap/scss/vendor/rfs";
```

Comment:
Why this import? Bootstrap already imports this with `import "bootstrap"` right?

Reply:
This import gives dev access to the Bootstrap specific rfs mixin `font-size`. Although rfs is included in the build its mixins are not is not in the dev environment.